protocol.md

```diff
@@ -1056,6 +1095,12 @@ export interface TextDocumentSyncOptions {
 
 interface ServerCapabilities {
 	/**
+	 * The server supports $/partialResult notifications.
+	 * If true, the client is allowed to let the result in the final response be null
+	 * and only send the result through $/partialResult
+	 */
+	streaming?: boolean;
```

Comment:
Does the server really need to declare support or not?

It seems like the client can declare support and then the server can choose to stream responses or not.